### PR TITLE
Math Magicians: refactor with hooks

### PR DESCRIPTION
**Pull request by developertariq**
**- Refactored the Calculator component (components/Calculator.js) from a class based component to a functional component with hooks.**

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -1,55 +1,49 @@
...
-    super(propbs);
-    this.state = {
-      value: null,
-      next: null,
-      operation: null,
-    };
-    this.handleClick = this.handleClick.bind(this);
-  }
+export default function Calculator() {
+  const initState = {
```
Comment (codepantha) [inline src/components/Calculator.js:5]: - [ ] [OPTIONAL] Please consider using ES6 arrow function syntax as it is important to start adopting the newest changes and improvements in the JavaScript language in order to write better code 👍

**Review by codepantha — APPROVED** [review]
# STATUS: APPROVED 🟢 ✔️ 

## Hi @developertariq 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights 🎯 
✔️ Descriptive PR
✔️ Refactored to use Hooks 📌 
✔️ Good commit history
✔️ No linter errors

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ] Well done for using the current README template 👏 Please edit your README with more information about your project as it is the first thing anyone who visits this repo sees and you want it to be as professional as possible 👍 
- [ ] Kindly add a MIT license to your project as it is advisable that every software (opensource included) has a license to tell developers how to use and contribute to your code 😎 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

## Required Changes ♻️
_Check the comments under the review._

File: src/components/Calculator.js (hunk 1 above)
Comment (developertariq) [reply src/components/Calculator.js:5]: Hi @codepantha,
Thanks for your suggestion. I hope you will find the changes in the next commit.

Comment (developertariq) [reply src/components/Calculator.js:5]: Hi @codepantha,
I changed the repository according to your suggestion. Would you please suggest to me what can I do? Merge the new branch or you want to review.

Thanks.

**Pull request merged**